### ISSUE TITLE
namespace import tincr gives name clashes

### ISSUE THREAD

**Issue by nelsobe** [bradselw/tincr#66]
The README.md file suggests you run namespace import ::tincr::* to import the Tincr commands into the global namespace.  When doing so, now gives error:

    Can't import command lremove: already exists

We should track this down down fix for future users.

**Comment by DallonTG** [comment]
The command that it couldn't import was actually "lremove".

**Comment by nelsobe** [comment]
Score one more for auto-correct.

**Comment by DallonTG** [comment]
The problem is that `lremove` is defined in both `tcl.tcl` and `util.tcl`.

tcl.tcl has `namespace import ::tcl::*` at the end of it, so by the time you try to import the tincr namespace, there is already a `lremove` command.

I don't see any need for the `lremove` command defined in `tcl.tcl` and no code seems to be using it. I vote we just remove it unless anyone knows of any issues it might cause.

Edit: 
Once I removed `lremove` from `tcl.tcl`, I got a similar error with the `suffix` command. The same suffix procedure was defined in both `util.tcl` and `family_info.tcl`. I removed the one in `family_info.tcl` and updated the code to use the procedure in `util.tcl`. After this, the namespace imported without errors.
